src/js/utils/underscore.d.ts

```diff
@@ -0,0 +1,6148 @@
+// Type definitions for Underscore 1.9
```

Comment:
Since we use a custom underscore lib file, this file is the type definition for it.
I just took the type definition file for the os underscore lib and added one of our custom methods.
I'm assuming there are a lot more, but I honestly did not take the time to see how many custom functions we have.
You may need to add the custom method type definition to this file if you try to use one.
However, if we wind up typing the underscore.js file, or upgrade and use the os one, we can get rid of this.

Reply:
is there a way to include this as a dependency rather than committing it to the repo?